bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -224,6 +224,22 @@
 		<state pattern="%s" readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="obstruction_alarm">
+		<item-type>Contact</item-type>
+		<label>Obstruction Alarm</label>
+		<description>Obstruction alarm of the blinds. The blinds were stopped and moved a bit in the opposite direction.</description>
+		<category>Blinds</category>
```

Comment:
"Blinds" looks not really appropriate to that particular channel type. I don't think any category should be set there.

Reply:
Done.